### DISCUSSION TITLE
在页面上屏蔽掉某些库

### DISCUSSION THREAD

**Discussion by qinzhi1209** [hhyo/Archery#2210] [General]
默认屏蔽了test  information_schema这些库，想添加一些自定义的库在哪里配置

**Comment by nick2wang** [comment]
可以在代码里自定义
https://github.com/hhyo/Archery/blob/cdbd12e21e1dae834365c078004cf156228f17a2/sql/engines/mysql.py#L151-L162

**Reply by qinzhi1209** [reply]
好的，谢谢
